server/jetty/src/main/java/io/deephaven/server/jetty/JettyConfig.java

```diff
@@ -133,6 +133,15 @@ public int port() {
     @Nullable
     public abstract Boolean http1();
 
+    /**
+     * Include sniHostCheck.
+     */
+    @Nullable
+    @Default
+    public Boolean sniHostCheck() {
+        return true;
```

Comment:
Please set up a config property to read from, rather than requiring a custom main.

Example use case: when testing reconnect behavior with h2 from another computer, custom self-signed certificates have to be created with local network names, rather than just telling the server to not care about SNI.